ipn/localapi/localapi.go

```diff
@@ -1200,6 +1201,24 @@ func (h *Handler) serveDial(w http.ResponseWriter, r *http.Request) {
 	<-errc
 }
 
+func (h *Handler) serveDeviceToken(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
+		http.Error(w, "unsupported method", http.StatusMethodNotAllowed)
+		return
+	}
```

Comment:
need permission checks here. See PermitWrite elsewhere in this file.

Is this only for setting the token? "set-device-token" and "serveSetDeviceToken"? Or will you add GET later so the frontend can decide whether it needs to do the init.